lib/elixir_auth_google.ex

```diff
@@ -8,33 +8,60 @@ defmodule ElixirAuthGoogle do
   @google_token_url "https://oauth2.googleapis.com/token"
   @google_user_profile "https://www.googleapis.com/oauth2/v3/userinfo"
 
+
+  @doc """
+  `get_baseurl_from_conn/1` derives the base URL from the conn struct
+  """
+  @spec get_baseurl_from_conn(Map) :: String.t
+  def get_baseurl_from_conn(%{host: h, port: p}) when host == "localhost" do
```

Comment:
sorry I did a typo on my first suggestion, it should be:

```suggestion
  def get_baseurl_from_conn(%{host: h, port: p}) when h == "localhost" do
```

Reply:
Yeah, I (_also_) only saw that after I had committed your suggestion ... 🙄 
I've fixed it on `localhost` and thanks to the `pre-commit` hook, I know the tests pass. 😉